doc/api/util.md

```diff
@@ -254,6 +254,21 @@ intended as a debugging tool. Some input values can have a significant
 performance overhead that can block the event loop. Use this function
 with care and never in a hot code path.
 
+## util.formatWithOptions(inspectOptions, format[, ...args])
+<!-- YAML
+added: REPLACEME
+-->
+
+This function is identical to [`util.format()`][], except in that it takes
+an `inspectOptions` argument which specifies default options for
```

Comment:
Question: `"default"` as in "unless overriden by custom inspect functions"?

Reply:
Yeah, I think `default` can be dropped here. :) It’s not really overridden unless it’s a format string with `%o` in it, but we already document that.